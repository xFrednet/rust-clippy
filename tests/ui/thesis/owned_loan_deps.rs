//@rustc-env: CLIPPY_PETS_PRINT=1

fn extend_string(data: &String) -> &String {
    data
}

#[forbid(clippy::borrow_pats)]
fn call_extend_simple() {
    let data = String::new();

    extend_string(&data).is_empty();
}

fn main() {}
