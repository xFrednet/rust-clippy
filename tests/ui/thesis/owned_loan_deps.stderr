TODO: implement analysis for named refs
